File: src/main/java/com/primogemstudio/advancedfmk/render/FilterTypes.java
package com.primogemstudio.advancedfmk.render;

import com.primogemstudio.advancedfmk.render.filter.FilterType;
import net.minecraft.resources.ResourceLocation;

import static com.primogemstudio.advancedfmk.AdvancedFramework.MOD_ID;

public class FilterTypes {
    public static final FilterType GAUSSIAN_BLUR = new FilterType(new ResourceLocation(MOD_ID, "gaussian_blur"));
    public static final FilterType FAST_GAUSSIAN_BLUR = new FilterType(new ResourceLocation(MOD_ID, "fast_gaussian_blur"));
}

File: src/main/java/com/primogemstudio/advancedfmk/render/FilterTypes.kt
package com.primogemstudio.advancedfmk.render

import com.primogemstudio.advancedfmk.AdvancedFramework
import com.primogemstudio.advancedfmk.render.filter.FilterType
import net.minecraft.resources.ResourceLocation

object FilterTypes {
    @JvmField
    val GAUSSIAN_BLUR = FilterType(ResourceLocation(AdvancedFramework.MOD_ID, "gaussian_blur"))
    @JvmField
    val FAST_GAUSSIAN_BLUR = FilterType(ResourceLocation(AdvancedFramework.MOD_ID, "fast_gaussian_blur"))
}
